mara_pipelines/parallel_tasks/files.py

```diff
@@ -52,21 +53,32 @@ def __init__(self, id: str, description: str, file_pattern: str, read_mode: Read
 
         self.target_table = target_table
         self._db_alias = db_alias
+        self._storage_alias = storage_alias
+        self.__storage_client = None
         self.timezone = timezone
 
     @property
     def db_alias(self):
         return self._db_alias or config.default_db_alias()
 
+    @property
+    def storage_alias(self):
+        return self._storage_alias or config.default_storage_alias()
+
+    @property
+    def _storage_client(self):
```

Comment:
Why do we need this state? All calls are basically done via commandline clients, so when keep it open here? Just for the iterate and the path_last_modification_timestamp ?